DEVELOPMENT.md

```diff
@@ -53,14 +33,30 @@ Make sure you have npm (node package manager) available on your terminal as well
 Install `phoenix` in development mode (using the `-e` flag) and with development dependencies (using the `[dev]` extra) by running
 
 ```bash
-pip install -e ".[dev]"
+pip install -e ".[dev,experimental]"
 ```
 
 from the repository root.
 
-## Running Scripts with `hatch`
+If you are working on our LLM orchestration framework integrations, you may also wish to install LlamaIndex or LangChain, in which case, you can add `llama_index` or `langchain` to the above command (e.g., `pip install -e ".[dev,experimental,llama_index]"`). Alternatively, you can install these packages from source.
+
+To install LlamaIndex from source,
```

Comment:
shouldn't we just add these to our `dev`, in which case this section would not be necessary?

Reply:
yup

Reply:
addressed in [a318af8](https://github.com/Arize-ai/phoenix/pull/1450/commits/a318af8ffdb0b3322f40f700e149b9bbf80a72b1)